src/Raven.Studio/typescript/viewmodels/resources/createDatabase.ts

```diff
@@ -410,37 +443,85 @@ class createDatabase extends dialogViewModelBase {
             });
         
         generalUtils.delayedSpinner(this.spinners.databaseLocationInfoLoading, task);
+    }  
+
+    updateFolderPathOptions() {
+        switch (this.databaseModel.restore.source()) {
+            case 'local':
+                this.updateLocalFolderPath(this.databaseModel.restore.localServerCredentials().backupDirectory())
+                    .done(result => this.folderPathOptions(result.List));
```

Comment:
What is the difference between ```folderPathOptions``` and ```backupFolderPathOptions```

Reply:
Removed `folderPathOptions`  - was duplicated